backend/app/services/custom_scraper.py

```
@@ -0,0 +1,454 @@
+        if not date_str:
+            return datetime.utcnow()
+        
+        try:
+            # ISO形式の場合
+            if 'T' in date_str:
+                return datetime.fromisoformat(date_str.replace('Z', '+00:00'))
+            
+            # RSSの日付形式
+            from email.utils import parsedate_to_datetime
+            return parsedate_to_datetime(date_str)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)